astropy/cosmology/core.py

```diff
@@ -777,6 +791,46 @@ def nu_relative_density(self, z):
 
         return prefac * self._neff_per_nu * rel_mass
 
+    def _get_nufunc_scalar(self):
+        """ Return the version of the nu relative density function
+        to use in integrals."""
+
+        if self._massivenu:
+            return self._massivenu_relative_density_scalar
+        else:
+            return lambda x : 0.22710731766 * self._Neff
```

Comment:
Yes, it is a joke. But then again, I would bet my money on cosmologists having Fortran compilers that I don't even know still exist anymore. I took a Cosmology class once, and I almost had to write N-body codes in Fortran; It was a very close call.